internal/configs/validate/validator.go

```diff
@@ -26,6 +28,24 @@ func New() Validator {
 
 type ConfigValidator struct{}
 
+// invalidConfig type is needed to make any error returned from Validator
+// to be ErrInvalidConfig.
+type invalidConfig struct {
+	err error
+}
+
+func (e *invalidConfig) Error() string {
+	return ErrInvalidConfig.Error() + ": " + e.err.Error()
+}
+
+func (e *invalidConfig) Is(err error) bool {
+	return err == ErrInvalidConfig
```

Comment:
Linter complains

```
comparing with == will fail on wrapped errors. Use errors.Is to check for a specific error (errorlint)
```

but as it *is* already `Is` method, following the recommendation will result in ∞ recursion ☺

Reply:
I was probably wrong about infinite recursion, but the direct comparison here is entirely correct, as the argument is not supposed to be unwrapped (as far as I understand).

Added a nolint annotation, filed https://github.com/polyfloyd/go-errorlint/issues/11